src/Command/InstallCommand.php

```diff
@@ -67,19 +67,26 @@ protected function configure(): void
             $plugin = getenv('CI_BUILD_DIR') !== false ? getenv('CI_BUILD_DIR') : null;
         }
 
+        // Add more options mapped to environment variables.
+        $dbUser = getenv('DB_USER') !== false ? getenv('DB_USER') : null;
+        $dbPass = getenv('DB_PASS') !== false ? getenv('DB_PASS') : '';
+        $dbName = getenv('DB_NAME') !== false ? getenv('DB_NAME') : 'moodle';
+        $dbHost = getenv('DB_HOST') !== false ? getenv('DB_HOST') : 'localhost';
+        $dbPort = getenv('DB_PORT') !== false ? getenv('DB_PORT') : '';
+
         $this->setName('install')
             ->setDescription('Install everything required for CI testing')
             ->addOption('moodle', null, InputOption::VALUE_REQUIRED, 'Clone Moodle to this directory', 'moodle')
             ->addOption('data', null, InputOption::VALUE_REQUIRED, 'Directory create for Moodle data files', 'moodledata')
             ->addOption('repo', null, InputOption::VALUE_REQUIRED, 'Moodle repository to clone', $repo)
             ->addOption('branch', null, InputOption::VALUE_REQUIRED, 'Moodle git branch to clone, EG: MOODLE_29_STABLE', $branch)
             ->addOption('plugin', null, InputOption::VALUE_REQUIRED, 'Path to Moodle plugin', $plugin)
-            ->addOption('db-type', null, InputOption::VALUE_REQUIRED, 'Database type, mysqli, pgsql or mariadb', $type)
-            ->addOption('db-user', null, InputOption::VALUE_REQUIRED, 'Database user')
-            ->addOption('db-pass', null, InputOption::VALUE_REQUIRED, 'Database pass', '')
-            ->addOption('db-name', null, InputOption::VALUE_REQUIRED, 'Database name', 'moodle')
-            ->addOption('db-host', null, InputOption::VALUE_REQUIRED, 'Database host', 'localhost')
-            ->addOption('db-port', null, InputOption::VALUE_REQUIRED, 'Database port', '')
+            ->addOption('db-type', null, InputOption::VALUE_REQUIRED, 'Database type, mysqli, pgsql or mariadb (DB env var)', $type)
+            ->addOption('db-user', null, InputOption::VALUE_REQUIRED, 'Database user (DB_USER env var)', $dbUser)
```

Comment:
Not sure about env var mention in brackets, what is the purpose if it? Do we need to mention CI_BUILD_DIR, MOODLE_REPO, MOODLE_BRANCH, etc, for consistency?

Reply:
Yeah,

I had the same debate with myself the other day. Just it was not clear how to document them, and finally decided to add that information to the command options, so it's documented @ https://moodlehq.github.io/moodle-plugin-ci/CLI.html#install 

But, yes, maybe they are not needed there, because others aren't. And, as far as they are commented in the dist and explained files... surely that's enough.

I'm going to remove them from the command description, leaving them as they were. Agree.